src/Cairo.jl

```diff
@@ -311,10 +323,10 @@ end
 
 function write_to_png(surface::CairoSurface, filename::AbstractString)
     ccall((:cairo_surface_write_to_png,_jl_libcairo), Void,
-          (Ptr{UInt8},Ptr{UInt8}), surface.ptr, bytestring(filename))
+          (Ptr{UInt8},Ptr{UInt8}), surface.ptr, @compat(String(filename)))
 end
 
-writemime(io::IO, ::MIME"image/png", surface::CairoSurface) =
+@compat Base.show(io::IO, ::MIME"image/png", surface::CairoSurface) =
```

Comment:
Something was going wrong with the webhook, not sure what. AppVeyor doesn't always handle repository moves all that smoothly. Collaborator or admin access would be fine - I think you can disable the emails somewhere in their settings, but it doesn't seem to obey disabling email notifications from the yml as well as travis does.